### ISSUE TITLE
[Anon] Bug report: Epic.gstz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-epic-armageddon#24]
**File:** Epic.gstz

**BattleScribe version:** 2.03.21

**Platform:** Windows

**Dropbox:** No

**Description:** When I click on the "View roster" button (eye icon), the roster gets many "BR>" inside every table output and we do not see line returns where they supposed to be.

**Comment by Freecode73** [comment]
Screenshot of the problem
![image](https://user-images.githubusercontent.com/13176309/123484965-32afd080-d5d7-11eb-8697-29acf0f2c258.png)

**Comment by afraeve** [comment]
The `<BR>` tag used to make a line break, so the example you're looking would appear neatly. It was really useful for the more complicated profiles.
Sadly, the BattleScribe developer removed that functionality when 40k 8th ed was released, probably because of the new "keywords" in that game that use the <> symbols. He's stated that he might include some other markup text in future that I can replace them with, but until then, I don't know what else to do.
#20

**Comment by Freecode73** [comment]
It is possible to change the presentation like 40k? They do not need line returns in their roster.

**Comment by afraeve** [comment]
40k works differently - there are separate profiles for the units & their weapons - and when a weapon has 2 stat lines in game, the team have to build it with 2 profiles in BattleScribe - so something like the Baneblade in your example would be 6 separate profiles. Multiplied over the GW, EpicUK, NetEA and F-ERC lists, that becomes quite unmanageable. To top all that off, when I built the very first file, I realised that even a simple weapon can have different values within a single codex...
BattleScribe just wasn't really designed with Epic in mind, I guess.
